11.3/sass/style.scss

```diff
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+$primary-text: #2d3436;
+$white: #fff;
+$light-gray: #b2bec3;
+$dark-green: rgba(0,184,148,0.7);
+$light-green: rgba(0,184,148,0.4);
+
+
+html {
+  font-size: 62.5%;
+}
+body {
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 1rem;
+  background-color: rgba(143,245,241,1);
+  background: linear-gradient(45deg, rgba(143,245,241,1) 0%, rgba(143,238,114,1) 71%, rgba(143,238,114,1) 100%) fixed;
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#8ff5f1', endColorstr='#8fee72', GradientType=1 );
```

Comment:
ten filter jest dla wspierania IE6-9 - to juz w mojej opinii historyczne przegladarki, linear-gradient IE wspiera od wersji 10, wiec nie bawilbym sie we wspieranie jeszcze starszych :)
Jesli chodzi o takie wsparcie, to polagalbym po prostu na tym, co zrobi autoprefixer